scripts/conduit.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Conduit daemon
+
+[Service]
+Type=forking
```

Comment:
might be good to give it a workdir and run it under a specific user?

Reply:
@lyuboxa why the working dir? If conduit is set up in `/usr/bin`, does that matter?

Reply:
it needs to create the database file somewhere, it better be somewhere known, like /var/lib/conduit